apps/nextjs/next.config.mjs

```diff
@@ -2,10 +2,12 @@
  * Run `build` or `dev` with `SKIP_ENV_VALIDATION` to skip env validation.
  * This is especially useful for Docker builds and Linting.
  */
-// !process.env.SKIP_ENV_VALIDATION && (await import("./src/env.mjs"));
+!process.env.SKIP_ENV_VALIDATION && (await import("./src/env.mjs"));
 
 /** @type {import("next").NextConfig} */
 const config = {
+  /** For docker build, more information: https://nextjs.org/docs/advanced-features/output-file-tracing */
+  output: "standalone",
```

Comment:
put this as part of the README docs instead. i dont want this to be the default resort